src/lem/pointers.rs

```diff
@@ -13,7 +15,7 @@ use super::Tag;
 /// children a pointer has. However, LEMs require extra flexibility because LEM
 /// hashing operations can plug any tag to the resulting pointer. Thus, the
 /// number of children have to be made explicit as the `Ptr` enum.
-#[derive(Clone, Copy, Debug, PartialEq, Eq)]
+#[derive(Clone, Copy, Debug, PartialEq, Eq, Serialize, Deserialize)]
 pub enum Ptr<F: LurkField> {
     Leaf(Tag, F),
```

Comment:
`Leaf` is a funny legacy name. I think these are expressions (based on their type) or maybe atoms(?) based on not being compound.

Reply:
I will change it to `Atom`